src/Utils/StyleBuilder.php

```diff
@@ -94,7 +94,7 @@ public function compile(?DOMElement $styleElement): ?string
         if ($styleElement->hasAttribute('scoped')) {
             $this->hasScoped = true;
             preg_match_all(
-                '/(?:^|\s)\s*[^@\s][\.a-z0-9-_:,\/\s]+?\s*[{]/i',
+                '/(?:^|\s)\s*[^@\s][.a-z0-9-_:,()\/\s]+?\s*[{]/i',
```

Comment:
I don't get it. Why the brackets? They are not brackets, they are just an empty capture group. If it matches for "(" and ")" as well, then not because of these but because you added the wildcard "." to the regex.
```suggestion
                '/(?:^|\s)\s*[^@\s][\.a-z0-9-_:,\(\)\/\s]+?\s*[{]/i',
```

Having a wildcard in there makes the whole list of options pointless.

Reply:
Between the square brackets, the charakters are what they are.
It's a list of characters where no backslash is needed.

Reply:
Ah, I didn't know about that at all. Shocked.